src/client/components/characters/CharacterEditor.tsx

```diff
@@ -390,8 +390,8 @@ function HPEditor({
       </div>
       <div>
         <small>
-          you can also edit hp by clicking on the hp in the healthbar of your
-          token on the map
+          You can also edit HP by clicking on the hp in the health bar of your
```

Comment:
```suggestion
          You can also edit HP by clicking on the HP in the health bar of your
```

Reply:
Good catch! I wonder if we should also change hp to HP everywhere in the code. AC is also spelled in caps for example.

Reply:
I personally am not a big fan of using all caps property names (like `AC`). I don't know why, but it looks odd to me, maybe because it looks like a constant. To be clear, `fooACBar`, `fooAcBar`, `ac`, `acFoo` all look fine to me, it is just `AC` (or potentially `HP`) that irritate me for some reason. But that might very well just be me of course :)

Reply:
Hmm, I like the all caps spelling but I also don't mind leaving it as is. Although I guess at the moment it is not consistent.
Anyone else have an opinion on this? @tom95 maybe?

Reply:
my preferred spelling I think is:

* addACNow
* ac = 5
* acFoo
* fooAC

so I suppose differentiating on whether or not it is used as the first word in the "camel-case-chain". Not sure if I'm making things worse with that suggestion though :D

Mixing upper and lower I dont like so much, as in "Ac"

Reply:
Okay, I think everything should be consistent now